### ISSUE TITLE
Photos uploaded by guests with Jetengine form

### ISSUE THREAD

**Issue by alican532** [CrocoBlock/suggestions#2650]
With the Jetengine form, the photos uploaded by the guests are not visible on the back or frontend.

You say the reason for this is security. But this should be our responsibility. I use recaptcha and prevent code execution from the media library with some security plugins. Please ensure that images uploaded by guests are available.

Thank you.

**Comment by alex-gorbonov** [comment]
need this too. would be very helpful!!

**Comment by mirajadesign** [comment]
Yes. This is basic functionality with Ninja Forms, Formidable Forms, Gravity Forms, etc. We need guest upload capability, please!

**Comment by Crocoblock** [comment]
I am closing this issue because it is a copy

Please keep an eye on [it](https://github.com/Crocoblock/suggestions/issues/7862)
